commit 02968aa55972b680fa00273188da281d7612ca3e
Author: Steve Dickson <steved@redhat.com>
Date:   Fri Sep 14 14:19:50 2007 -0400

    Changed the default paths in sm-notify to
    /var/lib/nfs/statd which is where the locking
    state is kept in Red Hat distros.
    
    Signed-off-by: Steve Dickson <steved@redhat.com>

diff --git a/utils/statd/sm-notify.c b/utils/statd/sm-notify.c
index 98c03f9..862889f 100644
--- a/utils/statd/sm-notify.c
+++ b/utils/statd/sm-notify.c
@@ -24,11 +24,13 @@
 #include <errno.h>
 #include <grp.h>
 
+#define STATD_PATH_XTN "statd/"
+
 #ifndef BASEDIR
 # ifdef NFS_STATEDIR
-#  define BASEDIR		NFS_STATEDIR
+#  define BASEDIR		NFS_STATEDIR "/" STATD_PATH_XTN
 # else
-#  define BASEDIR		"/var/lib/nfs"
+#  define BASEDIR		"/var/lib/nfs" "/" STATD_PATH_XTN
 # endif
 #endif
 
diff --git a/utils/statd/sm-notify.man b/utils/statd/sm-notify.man
index dd03b8d..69b7059 100644
--- a/utils/statd/sm-notify.man
+++ b/utils/statd/sm-notify.man
@@ -33,7 +33,7 @@ but this is configurable.
 For each NFS client or server machine to be monitored,
 .B rpc.statd
 creates a file in
-.BR /var/lib/nfs/sm ", "
+.BR /var/lib/nfs/statd/sm ", "
 and removes the file if monitoring is no longer required.
 .PP
 When the machine is rebooted,
@@ -47,7 +47,7 @@ Each machine has an
 which is basically an integer counter that is incremented
 each time the machine reboots. This counter is stored
 in
-.BR /var/lib/nfs/state ,
+.BR /var/lib/nfs/statd/state ,
 and updated by
 .BR sm-notify .
 .SS Security
@@ -149,13 +149,13 @@ By default,
 .B sm-notify
 forks and puts itself in the background after obtaining the
 list of hosts from
-.BR /var/lib/nfs/sm .
+.BR /var/lib/nfs/statd/sm .
 .SH FILES
-.BR /var/lib/nfs/state
+.BR /var/lib/nfs/statd/state
 .br
-.BR /var/lib/nfs/sm/*
+.BR /var/lib/nfs/statd/sm/*
 .br
-.BR /var/lib/nfs/sm.bak/*
+.BR /var/lib/nfs/statd/sm.bak/*
 .br
 .BR /var/run/sm-notify.pid
 .SH SEE ALSO
